### PR TITLE
Feature/595 Used MediaQuery to adjust the layout for all screens (upload_image_from_phone)

### LINKED ISSUE #595: Text overflow in "Vælg billede fra galleri" screen
**Describe the bug**
When the search bar is pressed in the "Vælg billede fra galleri" screen. There is an overflow in the text.

**To Reproduce**
1. Go to "Vælg billede fra galleri" screen
2. Click on the search bar.

**Expected behavior**
There should not be a pixel overflow.

**Actual behavior**
There is a pixel overflow

**Screenshots**
billede

**Environment (please complete the following information):**
- OS: Android
- Emulator: Yes
- WXGA 10.1 inch tablet
- APK Version [30]

### PR DESCRIPTION
**Pull request by ZohraAmini**
Closes #595 

Now it is possible to scroll in that screen  (upload_image_from_phone) so the textoverflow is eliminated.
It has been tested on (emulators) several devices such as:  mobile devices Pixel 2 and 3, Tablet (WXGA 10.1 API 30).

### CODE REVIEW

**Comment by Determinatoro** [comment]
Please use the following checklist, @twaxlol. Feel free to check it off as you go along.

**Code Design** 

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems     
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code does not introduce functionality that is unnecessary for solving the problem

**Code Readability** 

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no \"hack\", or obscure solution   

**Code Maintainability**

- [x] Has all the added code been properly commented

Only check one of the items below:

- [x] The functionality does **not** need to be covered by tests
- [ ] The functionality needs to be covered by tests

Only check these if the functionality needs to be covered by tests:

- [ ] All functionality is covered by tests
- [ ] The tests has sensible names
- [ ] By reading the tests you know what they cover
- [ ] The tests cover sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing

**Functionality**

For this part, open the app, and test:
- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

**Comment by Determinatoro** [comment]
Please use the following checklist, @Mustafa-ALD. Feel free to check it off as you go along.

**Code Design** 

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems     
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code does not introduce functionality that is unnecessary for solving the problem

**Code Readability** 

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no \"hack\", or obscure solution   

**Code Maintainability**

- [x] Has all the added code been properly commented

Only check one of the items below:

- [x] The functionality does **not** need to be covered by tests
- [ ] The functionality needs to be covered by tests

Only check these if the functionality needs to be covered by tests:

- [ ] All functionality is covered by tests
- [ ] The tests has sensible names
- [ ] By reading the tests you know what they cover
- [ ] The tests cover sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing

**Functionality**

For this part, open the app, and test:
- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

**Comment by twaxlol** [comment]
Er du muligt at lave en Flutter test, for at sikrer at der ikke sker et overflow?
Kan se at pubspec og environments json er pushed med, er dette meningen?
Ellers ser det godt ud

**Comment by ZohraAmini** [comment]
@twaxlol 
1. Med Flutter tests - mener du så unit tests? Hvis ja, så er det ikke nødvendigt i dette tilfælde, da det er kun UI ændringer. Det skal bare testes manuelt på emulatorer. 
2. pubspec og environment push er vist ikke meningen.

File: .gitignore [hunk 1]
```diff
@@ -685,3 +685,4 @@ android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java
 ios/Flutter/Flutter.podspec
 
 ios/Flutter/flutter_export_environment.sh
+assets/environments.json
```
Comment (Mustafa-ALD) [inline .gitignore]: Looks good it works and has no problem.
But I'm not sure that adding the environments.json file is part of the fix 🤔

**Review by Mustafa-ALD — CHANGES_REQUESTED** [review]

**Review by twaxlol — DISMISSED** [review]

File: .gitignore (hunk 1 above)
Comment (ZohraAmini) [reply .gitignore]: @Mustafa-ALD  request for changes has been acted upon (made a commit revert)

**Review by Mustafa-ALD — APPROVED** [review]
Look good

**Review by twaxlol — APPROVED** [review]

**Pull request merged**